specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/stable/2021-10-01/examples/ConnectedClustersListClusterCredentialResultNonAad.json

```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "subscriptionId": "1bfbb5d0-917e-4346-9026-1d3b344417f5",
+    "resourceGroupName": "k8sc-rg",
+    "clusterName": "testCluster",
+    "api-version": "2021-10-01",
+    "Properties": {
+      "authenticationMethod": "Token",
+      "clientProxy": true
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "kubeconfigs": [
```

Comment:
same as before: HybridConnectionConfig object should be there where CSP flag is true. We should have one example without CSP flag also.

Reply:
Added it